packages/smooth_app/lib/l10n/app_en.arb

```diff
@@ -273,7 +273,7 @@
     "@contribute_to_get_rewards": {
         "description": "Button description shown on a product, clicking the button opens a card with unanswered product questions, users can answer these to contribute to Open food facts and gain rewards."
     },
-    "question_sign_in_text": "Sign in to your Open Food Facts account to claim your contribution and to increase your creditability",
+    "question_sign_in_text": "Sign in to your Open Food Facts account to claim your contribution and to increase your credibility",
```

Comment:
I have a problem with the wording. You don't contribute to Open Food Facts to gain credibility.
"Sign in to your Open Food Facts account to get credit for your contribution" would be enough, according to me

Reply:
Applied 👍🏼